bindata/bootkube/config/config-overrides.yaml

```diff
@@ -6,9 +6,6 @@ kubeletClientInfo:
   keyFile: /var/run/secrets/kubelet-client/tls.key
 serviceAccountPublicKeyFiles:
 - /var/run/configmaps/sa-token-signing-certs/ca-bundle.crt
-servingInfo:
```

Comment:
where are these set now? in defaultconfig.yml?

Reply:
> where are these set now? in defaultconfig.yml?

the bootstrap has another override file that already has these.